keps/sig-node/127-user-namespaces/README.md

```diff
@@ -803,6 +899,10 @@ logs or events for this purpose.
 
 ###### How can someone using this feature know that it is working for their instance?
 
+If the runtime doesn't support user namespaces an error is returned by the kubelet.
```

Comment:
So the pod will effectively transition to Failed state, right?

Reply:
Yes, there is a FailedCreatePodSandBox event. kubectl is shown as
ContainerCreating and the pod events shows:

```
  Warning  FailedCreatePodSandBox  12s (x23 over 5m6s)  kubelet            Failed to create pod sandbox: user namespaces is not supported by the runtime
```